packages/amplify_authenticator/lib/src/l10n/input_resolver.dart

```diff
@@ -364,9 +364,19 @@ class InputResolverKey {
       inputResolver.resolve(context, this);
 }
 
+/// The resolver class for input widgets.
+///
+/// This class is responsible for resolving text to display
+/// in input field titles, as well as hints, warnings and other
+/// guidance for the end-user.
 class InputResolver extends Resolver<InputResolverKey> {
   const InputResolver();
 
+  /// Returns the [String] that is displayed as the input title.
+  ///
+  /// In addition to displaying the input title, this method is
+  /// used by the default implmentations of the [hint, empty and format] methods
```

Comment:
```suggestion
  /// used by the default implmentations of the [hint], [empty], and [format] methods
```

Reply:
Yeah - wasn't sure on the preferred style here. Thanks!